play-services-core/src/main/AndroidManifest.xml

```diff
@@ -485,7 +507,7 @@
                 android:resource="@string/gms_settings_summary" />
         </activity-alias>
 
-        <!-- Separate launcher icon from main activity so it can be disabled manualy -->
+        <!-- microG Settings activity-alias shown in Launcher -->
         <activity-alias
             android:enabled="true"
```

Comment:
It will be interesting to try something like this:
`android:enabled="@bool/is_alias_enabled"`
then add `<bool name="is_alias_enabled">true</bool>` to **bools.xm**l
and create **../values-v23/bools.xml** with this

```
<?xml version="1.0" encoding="utf-8"?>
<!--
  ~ Copyright (C) 2020 microG Project Team
  ~
  ~ Licensed under the Apache License, Version 2.0 (the "License");
  ~ you may not use this file except in compliance with the License.
  ~ You may obtain a copy of the License at
  ~
  ~     http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing, software
  ~ distributed under the License is distributed on an "AS IS" BASIS,
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  -->

<resources>
    <bool name="is_alias_enabled">false</bool>
</resources>
```

So launcher icon should be always disabled Android 6.0 and newer.
Settings integration is available starting from this android version.

Reply:
> 
> So launcher icon should be always disabled Android 6.0 and newer. Settings integration is available starting from this android version.

Honeslty I think that can be still visible, is a shortcut, wothout go on settings>system>microg.
For the same reason I have installed an app to open update on lineageos.